sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -49,19 +129,30 @@
   <class name="std::vector<caf::SRTrueParticle>" />
   <class name="std::vector<caf::ParticleMatch>" />
 
-  <class name="caf::SRCRTHit" />
+  <class name="caf::SRCRTHit" ClassVersion="10">
+   <version ClassVersion="10" checksum="3162534166"/>
+  </class>
   <class name="std::vector<caf::SRCRTHit>" />
-  <class name="caf::SRCRTTrack" />
+
+  <class name="caf::SRCRTTrack" ClassVersion="10">
+   <version ClassVersion="10" checksum="2589421679"/>
+  </class>
   <class name="std::vector<caf::SRCRTTrack>" />
-  <class name="caf::SRCRTHitMatch" />
-  <class name="caf::SRCRTTrackMatch" />
-
-  <enum name="caf::Plane_t" />
-  <enum name="caf::Det_t" />
-  <enum name="caf::MCType_t" />
-  <enum name="caf::Wall_t" />
-  <enum name="caf::generator_" />
-  <enum name="caf::interaction_mode_" />
-  <enum name="caf::genie_status_" />
-  <enum name="caf::g4_process_" />
+
+  <class name="caf::SRCRTHitMatch" ClassVersion="10">
+   <version ClassVersion="10" checksum="1521467565"/>
+  </class>
+
+  <class name="caf::SRCRTTrackMatch" ClassVersion="10">
+   <version ClassVersion="10" checksum="2672302824"/>
+  </class>
+
+  <enum name="caf::Plane_t" ClassVersion="10"/>
+  <enum name="caf::Det_t" ClassVersion="10"/>
+  <enum name="caf::MCType_t" ClassVersion="10"/>
+  <enum name="caf::Wall_t" ClassVersion="10"/>
+  <enum name="caf::generator_" ClassVersion="10"/>
+  <enum name="caf::interaction_mode_" ClassVersion="10"/>
+  <enum name="caf::genie_status_" ClassVersion="10"/>
+  <enum name="caf::g4_process_" ClassVersion="10"/>
```

Comment:
I am not sure about the topic... I can notice that GenReflex has not reacted (i.e. did not add a checksum).
Also, `enum` are by default represented just by integers, so the schema evolution has not much to do there.
For sure if we change `myEnumItem = 1` to `myEnumItem = 2` ROOT will not change the values that were `1` into `2`.

It shouldn't be harmful either, so it's good to stay until ROOT complains.

Reply:
Yeah, ok. I was trying to look up briefly and found nothing useful immediately. I will leave it in, in case it's useful for scheme evolution type things.